test/unit/storage-component.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { AuthIdentity } from '@dcl/crypto'
+import { createStorageComponent } from '../../src/ports/storage/component'
+import { IStorageComponent } from '../../src/ports/storage/types'
+import { createTestIdentity, generateRandomIdentityId } from '../utils/test-identity'
+
+describe('when testing storage component', () => {
```

Comment:
WDYT about removing this global scope as it's defining that's testing the storage component?